WordPress/src/main/java/org/wordpress/android/ui/domains/management/DomainManagementActivity.kt

```diff
@@ -10,6 +10,7 @@ import dagger.hilt.android.AndroidEntryPoint
 import kotlinx.coroutines.flow.launchIn
 import kotlinx.coroutines.flow.onEach
 import org.wordpress.android.ui.ActivityLauncher
+import org.wordpress.android.ui.domains.management.DomainManagementViewModel.*
```

Comment:
Adding a minor suggestion fixing the `WildcardImport` Detekt issue

```suggestion
import org.wordpress.android.ui.domains.management.DomainManagementViewModel.ActionEvent
```

Reply:
Thank you!